buildDockerImage.sh

```diff
@@ -4,6 +4,6 @@
 ./gradlew clean installDist
 
 # build the Docker image that packages mantis-control-plane
-docker build -t netflixoss/mantiscontrolplaneserver .
+docker build -t dev/mantiscontrolplaneserver .
```

Comment:
why `dev`?

Reply:
So as to not confuse with the images on docker-hub when developing locally.

Reply:
Ok we could probably add a release script to docker hub in the future. 👍